code/game/turfs/simulated/shuttle.dm

```diff
@@ -62,4 +62,140 @@
 /turf/simulated/shuttle/floor4/vox	//Vox skipjack floors
 	name = "skipjack floor"
 	oxygen = 0
-	nitrogen = MOLES_N2STANDARD + MOLES_O2STANDARD
\ No newline at end of file
+	nitrogen = MOLES_N2STANDARD + MOLES_O2STANDARD
+
+// Actual buildable walls. Yey! Complete with ultra-snowflakey code.
+
+/turf/simulated/wall/shuttle
+	name = "wall"
+	icon = 'icons/turf/walls.dmi'
+	walltype = "shuttle"
+	icon_state = "shuttle0"
+	smooth = SMOOTH_FALSE // Use old smoothwall.
+	canSmoothWith = list(
+	/turf/simulated/wall/shuttle,
+	/obj/structure/window/full/shuttle,
+	/obj/machinery/door/airlock,
+	/obj/machinery/door/unpowered,
+	/obj/machinery/door/poddoor,
+	/obj/structure/shuttle)
+	var/fullcorner = 0
+
+/turf/simulated/wall/shuttle/proc/do_old_smooth()
```

Comment:
That's not feasible right now. The new smoothing system doesn't allow for the use of the corner turfs used in the shuttle turf sprites. In fact, the new smoothing system isn't compatible with shuttle wall sprites at all.

If it makes you feel any better, the WINDOWS use the new smoothing system